### ISSUE TITLE
Direct enter commas . Parsing error: Declaration or statement expected.

### ISSUE THREAD

**Issue by hengkx** [typescript-eslint/typescript-eslint#1781]
**Repro**
![bug](https://user-images.githubusercontent.com/8358236/77249615-73ef5d00-6c7d-11ea-8618-9d19fb20a25e.gif)
Direct enter commas 

Reopening the file works
```JS
module.exports = {
  extends: [
    'airbnb-typescript',
    'plugin:react/recommended',
    'plugin:import/typescript',
    'plugin:@typescript-eslint/recommended',
    'prettier/@typescript-eslint',
    'plugin:prettier/recommended',
    'plugin:jest/recommended',
    'plugin:promise/recommended',
    'plugin:compat/recommended'
  ],
  env: {
    browser: true,
    node: true
  },
  parser: '@typescript-eslint/parser',
  parserOptions: {
    ecmaVersion: 2020,
    sourceType: 'module'
  },
  rules: {
    // A temporary hack related to IDE not resolving correct package.json
    'import/no-extraneous-dependencies': 'off'
  },
  settings: {
    'import/resolver': {
      // See https://github.com/benmosher/eslint-plugin-import/issues/1396#issuecomment-575727774 for line below
      node: {},
      webpack: {
        config: require.resolve('./configs/webpack.config.eslint.js')
      }
    }
  }
};

```

```TS
// your repro code case
```

**Expected Result**

**Actual Result**

**Additional Info**

**Versions**

| package                     | version |
| --------------------------- | ------- |
| `@typescript-eslint/parser` | `2.24.0` |
| `TypeScript`                | `3.7.5` |
| `ESLint`                    | `6.8.0` |
| `node`                      | `12.16.0` |
| `npm`                       | `6.13.4` |

**Comment by bradzacher** [comment]
What's the issue here? This looks like a valid error to me, as a dangling command is a syntax error.

Why did you leave all of the issue template fields blank?
Please fill out _as much detail as possible_.

Help me to help you.
